src/promise.js

```diff
@@ -91,7 +91,7 @@ export async function promiseAllBatched<A, B>(
   }
 
   // initially, we schedule <batch> items in parallel
-  await Promise.all(Array(Math.min(batch, items.length)).fill().map(step));
+  await Promise.all(Array(1).fill().map(step));
```

Comment:
can we put back the previous code? the optimisation is important to me. typically when we have lot of inputs/outputs it's very important on mobile (that is async) to batch things.

on top of this, changing this implementation it impacting not just libcore but all work we want to 'parallelize'